index.js

```diff
@@ -54,7 +72,19 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  if(movies.length === 0){
+    return 0;
+  }
+
+  let avgSum = 0;
```

Comment:
Something like `ratingsSum` might be more appropriate here since this variable since it doesn't actually represent the average until you divide it by the array length.